RecoLocalFastTime/FTLClusterizer/interface/MTDArrayBuffer.h

```diff
@@ -103,6 +164,38 @@ void MTDArrayBuffer::set(const FTLCluster::FTLHitPos& pix, float energy, float t
   set(pix.row(), pix.col(), energy, time, time_error);
 }
 
```

Comment:
Hi,
I kept this just in case some other codes use it. But what do you mean by "a few vectors now mismatch in size"? .It seems that the size of `hitEnergy_vec`, `hitTime_vec` and `hitTimeError_vec` are the same before and after. 

Thanks.

Reply:
I meant that this set extends other vectors.
I'd naively expect that MTDArrayBuffer should remain SoA-like (all member arrays of the same size); but maybe that's not needed for a good reason?

Reply:
Hi,
I am sorry that I misunderstood what you meant. And I think all member arrays should have the same size in the code right?
Thanks.